target/linux/ipq40xx/patches-4.14/997-device_tree_cmdline.patch.patch

```diff
@@ -0,0 +1,12 @@
+--- a/drivers/of/fdt.c
++++ b/drivers/of/fdt.c
+@@ -1127,6 +1127,9 @@ int __init early_init_dt_scan_chosen(uns
+ 	p = of_get_flat_dt_prop(node, "bootargs", &l);
+ 	if (p != NULL && l > 0)
+ 		strlcpy(data, p, min((int)l, COMMAND_LINE_SIZE));
++	p = of_get_flat_dt_prop(node, "bootargs-append", &l);
++	if (p != NULL && l > 0)
++		strlcat(data, p, min_t(int, strlen(data) + (int)l, COMMAND_LINE_SIZE));
```

Comment:
Filename has to be fixed (and chunkey should be mentioned for the patch in the commit-message as it originates from the nbg6617)

Reply:
That hack/patch is an openwrt classic. You can find it 

https://github.com/openwrt/openwrt/blob/master/target/linux/generic/pending-3.18/997-device_tree_cmdline.patch

From what I know, the trouble with it is that this specific approach has been deprecated by upstream. And there have been multiple attempts at different variations... But so far nothing has stuck. The latest "idea" is to use a new property-modifier: "/append-property/" 

https://www.spinics.net/lists/devicetree/msg201832.html

But no idea if this will come to pass or not.